_config.yml

```diff
@@ -23,6 +23,11 @@ defaults:
       githubbranch: "master"
       docsbranch: "master"
       versions:
+        - fullversion: "v1.8.0"
+          version: "v1.8"
+          githubbranch: "release-1.8"
```

Comment:
set *githubbranch* to `v1.8.0`

Reply:
Done, but I don't understand what branch that is.

Reply:
It's the GitHub branch from the main Kubernetes repo. If you do a search in the docs, you can see, for example, that in `/docs/admin/authorization/abac.md`, there is a link that uses that variable: 

`[Complete file example](http://releases.k8s.io/{{page.githubbranch}}/pkg/auth/authorizer/abac/example_policy_file.jsonl)`